src/client/sgidLogin/actions/index.ts

```diff
@@ -0,0 +1,15 @@
+import { get } from '../../app/util/requests'
+import { GetReduxState } from '../../app/actions/types'
+
+const getAuthRedirectionUrl = () => (_getState: GetReduxState) => {
```

Comment:
```suggestion
const getAuthUrl = () => (_getState: GetReduxState) => {
```
having `redirection` in the function name might be misleading as the `redirect URL` or `callback URL` has a different meaning in OAuth 2.0.

this is an authorization URL as it redirects to the `/oauth/authorize` sgID QR code page.

Reply:
rayner brought up a good nit point.

Reply:
Ahh makes sense, thanks!